Makefile

```diff
@@ -0,0 +1,13 @@
+tmpdir := $(shell mktemp -d -t loopback-sandbox)
+
+# This target updates all the files in the `loopback-sandbox` repository while
+# keeping `README.md` and `Makefile` intact.
+.PHONY: selfupdate
+selfupdate:
+	@cp $(CURDIR)/README.md $(tmpdir)
+	@cp $(CURDIR)/Makefile $(tmpdir)
+	@rm -rf $(CURDIR)/*
+	@slc loopback
+	@cp $(tmpdir)/README.md $(CURDIR)/README.md
+	@cp $(tmpdir)/Makefile $(CURDIR)/Makefile
```

Comment:
I also don't think this will work when I do git checkout `package.json` (because I want the script section to say `npm run selfupdate` and the new scaffolded `package.json` won't have this filled in)